src/Tag/Tag.tsx

```diff
@@ -2,15 +2,15 @@ import React, { FC } from 'react'
 import styled from 'styled-components'
 import { MarginProps } from '../utils/space'
 
-import { Text } from '../Text'
+import { Text, DeprecatedTypo, Typo } from '../Text'
 import { Color, theme } from '../theme'
 import { Box } from '../Box'
 
 export type DefaultProps = {
   label: string
   className?: string
   color: Color
-  typo?: string
+  typo?: Typo | DeprecatedTypo
```

Comment:
I wasn't even aware that we could change the `typo` here
Since this component is bound to be deprecated, should we leave it at `string`?
I'm pretty sure that we don't want to be able to update the typo for the next version

Reply:
yeah fair enought